File: projects/web/go/gin/{{cookiecutter.project_slug}}/internal/api/ping.go
package api

import (
	"net/http"

	"github.com/gin-gonic/gin"
)

// Ping reports the version. Every API exposes it at the root.
func Ping(version string) gin.HandlerFunc {
	return func(c *gin.Context) {
		c.JSON(http.StatusOK, gin.H{"status": "ok", "version": version})
	}
}

File: web/python/fastapi/{{cookiecutter.project_slug}}/app/api/ping.py
"""Ping. Every API exposes it at the root."""

from fastapi import APIRouter

ping_router = APIRouter(tags=["Ping"])


@ping_router.get("/ping")
def ping() -> dict[str, str]:
    from app import __version__

    return {"status": "ok", "version": __version__}
